static/js/admin.js

```diff
@@ -1205,18 +1205,20 @@ function remove_assignment() {
         return;
     }
 
-    var obj = new XMLHttpRequest();
-    obj.open('POST', '/runestone/admin/removeassign/' + assignmentid, true);
-    obj.send(JSON.stringify({ assignid: 'assignmentid' }));
-    obj.onreadystatechange = function () {
-        if (obj.readyState == 4 && obj.status == 200) {
+    var url = '/runestone/admin/removeassign';
```

Comment:
Doesn't this need a terminating semicolon?

Reply:
Sorry, this should be for the next line (1209).

Reply:
; are not required, and we don't really have great style guidelines or linting of any of our python or javscript.  We can add that to the todo list after we get better test coverage.

Reply:
Sounds good.